### PR TITLE
Clean up other test to use suggestions of #984

### PR DESCRIPTION
**Pull request by tcnghia**

### CODE REVIEW

**Comment by tcnghia** [comment]
/assign @vagababov 

Just realized that the test I copy/paste from could benefit from same suggestions as well.

File: network/prober/prober_test.go [hunk 1]
```diff
@@ -302,44 +302,29 @@ func TestWithHostOption(t *testing.T) {
...
-			if ok != test.success {
-				t.Errorf("unexpected probe result: want: %v, got: %v", test.success, ok)
-			}
-			if err != nil && !test.expErr {
-				t.Errorf("Do() = %v, no error expected", err)
-			}
-			if err == nil && test.expErr {
-				t.Errorf("Do() = nil, expected an error")
+			if ok, _ := Do(context.Background(), network.AutoTransport, ts.URL, test.options...); !ok {
+				t.Error("unexpected probe result")
```
Comment (vagababov) [inline network/prober/prober_test.go]:
```suggestion
				t.Error("Unexpected probe result")
```

Comment (tcnghia) [reply network/prober/prober_test.go]: Done

**Pull request merged**